tests/release/releaseLib.go

```diff
@@ -0,0 +1,26 @@
+package common
+
+import (
+        . "github.com/onsi/ginkgo/v2"
+        . "github.com/onsi/gomega"
+        appservice "github.com/redhat-appstudio/application-api/api/v1alpha1"
+        "github.com/redhat-appstudio/e2e-tests/pkg/framework"
+)
+
+func CreateComponentByCDQ(devFw framework.Framework, devNamespace, managedNamespace, fbcAppName, fbcCompName string, sourceGitURL string) *appservice.Component {
```

Comment:
Can we just call it `appName` and `compName` so it isn't fbc specific?

Reply:
Updated. Thanks